Aptoide/aptoide-client-v8
# Feature/asv 558 get default preferences from settings

**What does this PR do?**

Initialises default values in shared preferences in the AptoideApplication class by parsing settings.xml

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Change values in settings.xml and make sure the changes are being correctly set in sharedpreferences

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-558](https://aptoide.atlassian.net/browse/ASV-558)


**Code Review Checklist**
